device-types/Eaton/9SX5KIRT.yaml

```diff
@@ -0,0 +1,56 @@
+---
+manufacturer: Eaton
+model: 9SX5KIRT
+slug: eaton-9sx5kirt
+u_height: 3
+is_full_depth: true
+comments: 9SX5KIRT 5kVA 230V Tower/Rack Mountable UPS [Manual](https://www.eaton.com/content/dam/eaton/products/backup-power-ups-surge-it-power-distribution/backup-power-ups/eaton-9px-ups/user-guides/eaton-9sx-9px-ups-5-6-kva-ebm-180v-installation-and-user-manual.pdf)
+power-outlets:
+  - name: Primary Group Hardwired
+    type: hardwired
+    power_port: AC Source
+  - name: Primary Group - Output 1
+    type: iec-60320-c19
+    power_port: AC Source
+  - name: Primary Group - Output 2
+    type: iec-60320-c19
+    power_port: AC Source
+  - name: Group 1 - Output 1
+    type: iec-60320-c13
+    power_port: AC Source
+  - name: Group 1 - Output 2
+    type: iec-60320-c13
+    power_port: AC Source
+  - name: Group 1 - Output 3
+    type: iec-60320-c13
+    power_port: AC Source
+  - name: Group 1 - Output 4
+    type: iec-60320-c13
+    power_port: AC Source
+  - name: Group 2 - Output 1
+    type: iec-60320-c13
+    power_port: AC Source
+  - name: Group 2 - Output 2
+    type: iec-60320-c13
+    power_port: AC Source
+  - name: Group 2 - Output 3
+    type: iec-60320-c13
+    power_port: AC Source
+  - name: Group 2 - Output 4
+    type: iec-60320-c13
+    power_port: AC Source
+power-ports:
+  - name: AC Source
+    type: hardwired
+interfaces:
+  - name: Battery connector
+    type: other
+  - name: Battery detection
+    type: other
```

Comment:
These are not meant for battery connectors, it is meant for network interfaces. These might be better defined as `module-bays`